src/test/java/org/isf/medicalstock/test/Tests.java

```diff
@@ -1194,6 +1196,13 @@ public void testMovementGetOrigin() throws Exception {
 		assertThat(movement).isNotNull();
 		assertThat(movement.getSupplier()).isEqualTo(movement.getOrigin());
 	}
+	
+	@Test
+	public void testDeleteLastMovement() throws Exception {
+		int code = setupTestMovement(false);
+		Optional<Movement> movement = movementIoOperationRepository.findById(code); 
+		assertAll(()-> movBrowserManager.deleteLastMovement(movement.get()));
```

Comment:
Doesn't `deleteLastMovement()` return `void`?  So what is being asserted here?  Wouldn't be better to do the delete and then try to fetch the same record which should then fail which can then be tested?   That is the pattern used in other Core tests FWIW.